front/app/containers/Admin/dashboard/messages.ts

```diff
@@ -124,7 +124,7 @@ export default defineMessages({
       'The number of participants that posted inputs, reacted or commented.',
   },
   inputsByStatusTitle: {
-    id: 'app.containers.AdminPage.DashboardPage.inputsByStatusTitle',
+    id: 'app.containers.AdminPage.DashboardPage.inputsByStatusTitle1',
```

Comment:
Why do we need to change these ids if we are not changing the copy?

Reply:
For some reasons (probably, because of a mistake during merge) the en.json still has "Posts" https://github.com/CitizenLabDotCo/citizenlab/blob/6b302838328f5f5d38040554e4b40a1acf29be87/front/app/translations/admin/en.json#L1033

Reply:
So, I change the ID to update the copy in `en.json`.